contracts/governance/Governor.sol

```diff
@@ -29,22 +28,22 @@ import "./IGovernor.sol";
 abstract contract Governor is Context, ERC165, EIP712, IGovernor, IERC721Receiver, IERC1155Receiver {
     using DoubleEndedQueue for DoubleEndedQueue.Bytes32Deque;
     using SafeCast for uint256;
-    using Timers for Timers.BlockNumber;
 
     bytes32 public constant BALLOT_TYPEHASH = keccak256("Ballot(uint256 proposalId,uint8 support)");
     bytes32 public constant EXTENDED_BALLOT_TYPEHASH =
         keccak256("ExtendedBallot(uint256 proposalId,uint8 support,string reason,bytes params)");
 
     struct ProposalCore {
-        Timers.BlockNumber voteStart;
-        Timers.BlockNumber voteEnd;
+        uint256 voteStart; // retyped from Timers.BlockNumber
+        uint256 voteEnd; // retyped from Timers.BlockNumber
         bool executed;
         bool canceled;
         address proposer;
```

Comment:
Now that I think of it we should be able to put this inside the unused space from the timers!

Reply:
Shouldn't that break backwards compatibility? We're changing the positions of variables in storage.

Reply:
Oh but this gets reset after executing. I'm worried about the order of deletion. For example, changing the slot first and then deleting the previous slot.

Where is that order?